independent-publisher-connectors/SchoolDigger/apiProperties.json

```diff
@@ -30,7 +30,18 @@
     },
     "iconBrandColor": "#da3b01",
     "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setqueryparameter",
+        "title": "App ID",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "appID",
+          "x-ms-apimTemplateParameter.value": "@connectionParameters('app_id')",
+          "x-ms-apimTemplateParameter.existsAction": "override"
+        }
+      }
+    ],
     "publisher": "Troy Taylor",
     "StackOwner": "SchoolDigger"

```

Comment:
@troystaylor one of the checks fails because of this property. Could you please change it to `"stackOwner"`?